pkg/network/driver_interface.go

```diff
@@ -189,29 +189,35 @@ func (di *DriverInterface) GetStats() (map[string]interface{}, error) {
 }
 
 // GetConnectionStats will read all flows from the driver and convert them into ConnectionStats
-func (di *DriverInterface) GetConnectionStats() ([]ConnectionStats, error) {
+func (di *DriverInterface) GetConnectionStats() ([]ConnectionStats, []ConnectionStats, error) {
 	readbuffer := make([]uint8, 1024)
-	connStats := make([]ConnectionStats, 0)
+	connStatsActive := make([]ConnectionStats, 0)
+	connStatsClosed := make([]ConnectionStats, 0)
 
 	for {
 		var count uint32
 		var bytesused int
 		err := windows.ReadFile(di.driverFlowHandle.handle, readbuffer, &count, nil)
 		if err != nil && err != windows.ERROR_MORE_DATA {
-			return nil, err
+			return nil, nil, err
 		}
 		var buf []byte
 		for ; bytesused < int(count); bytesused += C.sizeof_struct__perFlowData {
 			buf = readbuffer[bytesused:]
 			pfd := (*C.struct__perFlowData)(unsafe.Pointer(&(buf[0])))
-			connStats = append(connStats, FlowToConnStat(pfd))
+			if (pfd.flags & 0x10) == 0x10 {
```

Comment:
Is there a constant we can use here ?

Reply:
My instinct would be to set `0x10` to a constant in `ddfilter.c` and create macro to run on a flow's flags, similar to what we've done with direction. If we were doing this after converting the object to a ConnectionStat, I would suggest doing something in go, but since we're still using the C struct I think it makes more sense to do it this way. Thoughts?

Reply:
Do you mean `ddfilter.h` ? Sounds good to me!